etc/profile-m-z/xlinks.profile

```diff
@@ -1,21 +1,18 @@
 # Firejail profile for xlinks
 # Description: Text WWW browser (X11)
 # This file is overwritten after every install/update
+quiet
 # Persistent local customizations
 include xlinks.local
 # Persistent global definitions
-# added by included profile
-#include globals.local
+include globals.local
 
-noblacklist /tmp/.X11-unix
-noblacklist ${HOME}/.links
+noblacklist ${HOME}/.xlinks
 
-include whitelist-common.inc
+mkdir ${HOME}/.xlinks
+whitelist ${HOME}/.xlinks
 
-# if you want to use user-configured programs add 'private-bin PROGRAM1,PROGRAM2'
-# to your xlinks.local or append 'PROGRAM1,PROGRAM2' to this private-bin line
 private-bin xlinks
-private-etc fonts
 
 # Redirect
-include links.profile
+include links-common.profile
```

Comment:
How can "Text WWW browser (*X11*)" work with `blacklist /tmp/.X11-unix`?

Reply:
"links2 -g" worked... 
i'm on 32bit

Reply:
TCP sockets enabled? or is this because of abstract unix sockets? IDK but this PR removes `noblacklist /tmp/.X11-unix` from xlinks.profile.

Reply:
there's a potential error here

Reply:
"/tmp/.X11-unix" leeks somehow

Reply:
how do you mean this? what did you try to test?

Reply:
no, never mind
actually, it doesn't use the socket, you can navigate inside xlinks to /tmp/.X11-unix and it will say permission denied.